site/_diagrams/docs/admin-guide/blue_green_upgrade.plantuml

```diff
@@ -0,0 +1,22 @@
+@startuml
+  !include ../../_config.plantuml
```

Comment:
These don't render for me, as I don't have graphviz. Now that we have mermaid available, can you please do these as mermaid diagrams?